examples/new_project_templates/trainer_cpu_template.py

```diff
@@ -67,7 +67,7 @@ def main(hparams):
     add_default_args(parent_parser, root_dir)
 
     # allow model to overwrite or extend args
-    parser = ExampleModel.add_model_specific_args(parent_parser)
+    parser = LightningTemplateModel.add_model_specific_args(parent_parser)
```

Comment:
why do you prefer this model here?

Reply:
I am not aware of this change, let me check it...

Reply:
OK, the issue was that the `ExampleModel` was not imported so it was not clear where it appeared... :smile: